packages/core/src/wallet/IndyWallet.ts

```diff
@@ -34,32 +28,36 @@ export class IndyWallet implements Wallet {
     this.indy = agentConfig.agentDependencies.indy
   }
 
+  public get isCreated() {
```

Comment:
isCreated feels a bit misleading. I would assume this means whether the wallet was created in the current session, but if you call open it will also be set. 

Maybe we can call it something like `isProvisioned`. According to google the term provision means:

> the action of providing or supplying something for use.

Which is exactly what having the wallet config means: it has the needed info to use the wallet

Reply:
Agree. I also didn't like it so much.